src/show/components/Button/index.js

```diff
@@ -20,7 +21,8 @@ const Button = ({
   return (
     <button
       className={`flex px-2 transition-all duration-300 ${buttonColor[variant]} ${className}`}
```

Comment:
Does this button actually look like a disabled button if i set this to disabled={true}?